content/millennium/r4/financial/general/insurance-plan.md

```diff
@@ -0,0 +1,67 @@
+---
+title: InsurancePlan | R4 API
+---
+
+# InsurancePlan
+
+* TOC
+  {:toc}
+
+## Overview
+
+InsurancePlan describes a health insurance offering comprised of a list of covered benefits, costs associated with those benefits, and additional information about the offering, such as who it is owned and administered by, a coverage area contact information.
+
+The following fields are returned if valued:
+
+* [InsurancePlan id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status](https://hl7.org/fhir/r4/insuranceplan-definitions.html#InsurancePlan.status){:target="_blank"}
+* [Type](https://hl7.org/fhir/r4/insuranceplan-definitions.html#InsurancePlan.type){:target="_blank"}
+* [Name](https://hl7.org/fhir/r4/insuranceplan-definitions.html#InsurancePlan.name){:target="_blank"}
+* [Period](https://hl7.org/fhir/r4/insuranceplan-definitions.html#InsurancePlan.period){:target="_blank"}
+* [OwnedBy](https://hl7.org/fhir/r4/insuranceplan-definitions.html#InsurancePlan.ownedBy){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:insurance_plan, :r4) %>
+
+## Search
+
+Search for InsurancePlans that meet supplied query parameters:
+
+    GET /InsurancePlan?:parameters
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true, system: true) %>
+
+### Parameters
+
+Name                  | Required? | Type          | Description
+-----------------------|-----------|---------------|-------------------------------------------------------
+`owned-by`                 | Yes       | [`reference`]     | Organization associated with the InsurancePlan.
```

Comment:
Can we not align the headers with the field below, for better visibility?

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/809/commits/e01eb3f116a8cca3fcf44c0e072ca8816efa0134#diff-505b119a76d7cbd4b9ad19f605bcfd1d674bfb0fca50632ebc7948b1bc50b3caR40-R41